packages/vaadin-lumo-styles/src/components/message.css

```
@@ -0,0 +1,80 @@
+/**
+ * @license
+ * Copyright (c) 2000 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+:host {
+  display: flex;
+  flex-direction: row;
+  outline: none;
+  color: var(--lumo-body-text-color);
+  font-family: var(--lumo-font-family);
+  font-size: var(--lumo-font-size-m);
+  line-height: var(--lumo-line-height-m);
+  padding: var(--lumo-space-s) var(--lumo-space-m);
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+  -webkit-text-size-adjust: 100%;
+  --_focus-ring-color: var(--vaadin-focus-ring-color, var(--lumo-primary-color-50pct));
+  --_focus-ring-width: var(--vaadin-focus-ring-width, 2px);
+}
+
+:host([hidden]) {
+  display: none !important;
+}
+
+[part='content'] {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+}
+
+[part='header'] {
+  align-items: baseline;
+  display: flex;
+  flex-flow: row wrap;
+  min-height: calc(var(--lumo-font-size-m) * var(--lumo-line-height-m));
+}
+
+[part='name'] {
+  font-weight: 500;
+  margin-right: var(--lumo-space-s);
```

Would it make sense to use CSS logical properties here and remove the `:host([dir='rtl'])[part='name']` selector down in the file? Or are these PRs meant only to port the existing styles with no refactor in the process?

```suggestion
  margin-inline-end: var(--lumo-space-s);
```